src/components/needs-and-leads/NeedsTable.tsx

```diff
@@ -138,14 +139,14 @@ function NeedsTable(props: Props) {
                         color="primary"
                         onClick={() => console.log("clicked edit")}
                       >
-                        <EditIcon />
+                        <Edit />
                       </IconButton>
                       <IconButton
                         size="small"
                         color="primary"
                         onClick={() => console.log("clicked delete")}
                       >
-                        <DelIcon />
+                        <Backspace />
```

Comment:
same here, we don't know that it's an icon

Reply:
Ok